lib/pgBackRest/Expire.pm

```diff
@@ -224,24 +230,33 @@ sub process
         }
     }
 
```

Comment:
We talked about moving this section to configLoad().  I think that's a good idea to allow the values will show up in the expire start list.  That also means you could take out the debug output line 251.